pontoon/projects/management/commands/send_review_notifications.py

```diff
@@ -0,0 +1,83 @@
+from collections import defaultdict
+from datetime import timedelta
+from urllib.parse import urlencode
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from django.urls import reverse
+from django.utils import timezone
+from notifications.signals import notify
+from pontoon.base.models import Translation
+
+
+class Command(BaseCommand):
+    help = "Notify translators about their newly reviewed suggestions"
+
+    def get_description(self, author, notifyData):
+        desc = "Your suggestions have been reviewed:"
+
+        for (locale, project), (approved, rejected) in notifyData.items():
+            url = reverse(
+                "pontoon.translate",
+                kwargs={
+                    "locale": locale.code,
+                    "project": project.slug,
+                    "resource": "all-resources",
+                },
+            )
+            if len(approved) == 1 and len(rejected) == 0:
+                url += "?" + urlencode({"string": approved[0]})
+            elif len(approved) == 0 and len(rejected) == 1:
+                url += "?" + urlencode({"string": rejected[0]})
+            else:
+                url += "?" + urlencode({"author": author.email})
```

Comment:
I would suggest we filter by author in all cases. Or block this whole patch on #2049?

Reply:
Added author filter; that does make sense always. Still including the `&string=` when there's only one.

Recommend against blocking on #2049, because the solution to that is likely to be built on top of #2452, and that would require that whole stack of things to resolve before we get to merge this.